components/layout/header.vue

```diff
@@ -116,11 +138,11 @@ import {
   XIcon,
 } from '@heroicons/vue/outline'
 import { ChevronDownIcon } from '@heroicons/vue/solid'
-
+import { Icon } from '@iconify/vue'
 const navigation = [
-  { name: 'Find Accomodation', to: '/guest' },
-  { name: 'UAH Exchange', to: '/exchange' },
-  { name: 'Recommend', to: '/add' },
+  { name: 'Add service', to: '/guest' },
```

Comment:
Not sure that links(to:/) correspond to the names of menu items

Reply:
Yeah I told you that I need the links for those texts like 'Add service'